Maestro/src/Microsoft.DotNet.Maestro.WebApi/Controllers/CommitPushedController.cs

```diff
@@ -20,12 +24,22 @@ public class CommitPushedController : ApiController
         private EventRegistrationTable _eventService = new EventRegistrationTable();
         private AzureStorageService _storageService = new AzureStorageService();
 
-        public async Task Post(PushWebHookEvent e)
+        public async Task Post()
         {
+            // The PushWebHookEvent object must be manually deserialized instead of using parameter binding 
+            // because the payload signature verification requires access to the request's raw content.  When
+            // using parameter binding, the content is read during the binding and cannot be accessed again due
+            // to Web API treating the request's content as a forward-only stream which can only be read once.
```

Comment:
I'd like to understand why this GitHub webhook class can read the HttpRequestMessage.Content twice:

[https://github.com/aspnet/WebHooks/blob/master/src/Microsoft.AspNet.WebHooks.Receivers.GitHub/WebHooks/GitHubWebHookReceiver.cs](https://github.com/aspnet/WebHooks/blob/master/src/Microsoft.AspNet.WebHooks.Receivers.GitHub/WebHooks/GitHubWebHookReceiver.cs)

Once here:  [https://github.com/aspnet/WebHooks/blob/master/src/Microsoft.AspNet.WebHooks.Receivers.GitHub/WebHooks/GitHubWebHookReceiver.cs](https://github.com/aspnet/WebHooks/blob/master/src/Microsoft.AspNet.WebHooks.Receivers.GitHub/WebHooks/GitHubWebHookReceiver.cs)#L136

and another time here:  [https://github.com/aspnet/WebHooks/blob/master/src/Microsoft.AspNet.WebHooks.Receivers.GitHub/WebHooks/GitHubWebHookReceiver.cs](https://github.com/aspnet/WebHooks/blob/master/src/Microsoft.AspNet.WebHooks.Receivers.GitHub/WebHooks/GitHubWebHookReceiver.cs)#L74